2023-05-29 20:17:43,894 (40018ab808) :  INFO (core:349) - Starting framework core - Version: 2.6.3, Build: 248a2da (Thu May 18 13:03:39 UTC 2023)
2023-05-29 20:17:43,899 (40018ab808) :  DEBUG (core:361) - Using the elevated policy
2023-05-29 20:17:43,902 (40018ab808) :  DEBUG (core:450) - Starting runtime component.
2023-05-29 20:17:43,918 (40018ab808) :  DEBUG (core:450) - Starting caching component.
2023-05-29 20:17:43,920 (40018ab808) :  DEBUG (core:450) - Starting data component.
2023-05-29 20:17:43,921 (40018ab808) :  DEBUG (core:450) - Starting networking component.
2023-05-29 20:17:43,924 (40018ab808) :  DEBUG (networking:266) - No cookie jar found
2023-05-29 20:17:43,941 (40018ab808) :  DEBUG (networking:434) - Setting the default network timeout to 20.0
2023-05-29 20:17:43,946 (40018ab808) :  DEBUG (core:450) - Starting localization component.
2023-05-29 20:17:43,947 (40018ab808) :  INFO (localization:409) - Setting the default locale to en-us
2023-05-29 20:17:43,949 (40018ab808) :  DEBUG (core:450) - Starting messaging component.
2023-05-29 20:17:43,956 (40018ab808) :  DEBUG (core:450) - Starting debugging component.
2023-05-29 20:17:43,959 (40018ab808) :  DEBUG (core:450) - Starting services component.
2023-05-29 20:17:43,961 (400d380b38) :  DEBUG (networking:144) - Requesting 'http://127.0.0.1:32400/system/messaging/clear_events/tv.plex.agents.series'
2023-05-29 20:17:43,962 (40018ab808) :  DEBUG (core:450) - Starting myplex component.
2023-05-29 20:17:43,971 (40018ab808) :  DEBUG (core:450) - Starting notifications component.
2023-05-29 20:17:44,606 (40018ab808) :  DEBUG (accessor:68) - Creating a new model access point for provider tv.plex.agents.series in namespace 'metadata'
2023-05-29 20:17:44,636 (40018ab808) :  DEBUG (networking:144) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes'
2023-05-29 20:17:45,018 (40018ab808) :  ERROR (networking:197) - Error opening URL 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes'
2023-05-29 20:17:45,025 (40018ab808) :  CRITICAL (runtime:1299) - Exception getting hosted resource hashes (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-248a2daf0/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/runtime.py", line 1293, in get_resource_hashes
    json = self._core.networking.http_request("http://127.0.0.1:32400/:/plugins/com.plexapp.system/resourceHashes", timeout=10).content
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-248a2daf0/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 243, in content
    return self.__str__()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-248a2daf0/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 221, in __str__
    self.load()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-248a2daf0/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 159, in load
    f = self._opener.open(req, timeout=self._timeout)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 435, in open
    response = meth(req, response)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 548, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 473, in error
    return self._call_chain(*args)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 556, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 404: Not Found

2023-05-29 20:17:45,063 (400dcc7b38) :  DEBUG (services:265) - Plug-in is not daemonized - loading services from system
2023-05-29 20:17:45,063 (40018ab808) :  DEBUG (runtime:1117) - Created a thread named 'load_all_services'
2023-05-29 20:17:45,072 (40018ab808) :  DEBUG (runtime:1117) - Created a thread named 'get_server_info'
2023-05-29 20:17:45,074 (400dd14b38) :  DEBUG (networking:144) - Requesting 'http://127.0.0.1:32400'
2023-05-29 20:17:45,075 (400dcc7b38) :  DEBUG (networking:144) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/messaging/function/X0J1bmRsZVNlcnZpY2U6QWxsU2VydmljZXM_/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMApyMAo_'
2023-05-29 20:17:45,091 (40018ab808) :  DEBUG (core:150) - Finished starting framework core
2023-05-29 20:17:45,093 (40018ab808) :  DEBUG (core:560) - Loading plug-in code
2023-05-29 20:17:45,102 (40018ab808) :  DEBUG (core:566) - Finished loading plug-in code
2023-05-29 20:17:45,110 (40018ab808) :  DEBUG (agentkit:1132) - Creating new agent class called PlexSeries
2023-05-29 20:17:45,113 (40018ab808) :  DEBUG (agentkit:937) - Updating agent information: [{'media_types': ['TV_Show'], 'accepts_from': None, 'fallback_agent': None, 'contributes_to': None, 'languages': [], 'persist_stored_files': True, 'version': 0, 'primary_provider': True, 'prefs': True, 'name': 'Plex TV Series'}]
2023-05-29 20:17:45,125 (400dd14b38) :  DEBUG (core:538) - Machine identifier is 260f3b370065325658e9c678f9981104096370bb
2023-05-29 20:17:45,127 (400dd14b38) :  DEBUG (core:539) - Server version is 1.32.2.7100-248a2daf0
2023-05-29 20:17:45,128 (40018ab808) :  DEBUG (networking:144) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/messaging/function/X0FnZW50U2VydmljZTpVcGRhdGVJbmZv/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQo1CmRpY3QKbGlzdApkaWN0Cmxpc3QKbGlzdAoyCnMyMQp0di5wbGV4LmFnZW50cy5zZXJpZXNzMTAKaWRlbnRpZmllcnIxCnMxMAphZ2VudF9pbmZvMQpyMgoxMApyMwpzMTEKbWVkaWFfdHlwZXNuczEyCmFjY2VwdHNfZnJvbW5zMTQKZmFsbGJhY2tfYWdlbnRuczE0CmNvbnRyaWJ1dGVzX3RvcjQKczkKbGFuZ3VhZ2VzYjFzMjAKcGVyc2lzdF9zdG9yZWRfZmlsZXNpMApzNwp2ZXJzaW9uYjFzMTYKcHJpbWFyeV9wcm92aWRlcmIxczUKcHJlZnNzMTQKUGxleCBUViBTZXJpZXNzNApuYW1lMQpzNwpUVl9TaG93MApyMAo_'
2023-05-29 20:17:45,201 (400dcc7b38) :  DEBUG (services:362) - Loaded services
2023-05-29 20:17:45,223 (40018ab808) :  CRITICAL (sandbox:298) - Function named 'Start' couldn't be found in the current environment
2023-05-29 20:17:45,224 (40018ab808) :  INFO (core:611) - Started plug-in
2023-05-29 20:17:45,226 (40018ab808) :  DEBUG (socketinterface:160) - Starting socket server
2023-05-29 20:17:45,229 (40018ab808) :  DEBUG (runtime:1117) - Created a thread named 'start'
2023-05-29 20:17:45,233 (40018ab808) :  INFO (socketinterface:184) - Socket server started on port 36707
2023-05-29 20:17:45,234 (400dcf1b38) :  DEBUG (services:438) - No shared code to load
2023-05-29 20:17:45,235 (40018ab808) :  INFO (pipeinterface:25) - Entering run loop
2023-05-29 20:17:45,239 (40018ab808) :  DEBUG (runtime:717) - Handling request GET /:/prefixes
2023-05-29 20:17:45,246 (40018ab808) :  DEBUG (runtime:814) - Found route matching /:/prefixes
2023-05-29 20:17:45,252 (40018ab808) :  DEBUG (runtime:924) - Response: [200] MediaContainer, 146 bytes
